backend/benefit/common/tests/conftest.py

```diff
@@ -11,6 +13,7 @@
 @pytest.fixture(autouse=True)
 def setup_test_environment(settings):
     factory.random.reseed_random("777")
+    random.seed(777)
```

Comment:
is it leftover? Or what is the purpose of it?

Reply:
Had discussion about randomization of tests, yes this is leftover from that, removed